src/classify_reads.rs

```diff
@@ -0,0 +1,83 @@
+use std::str;
+use std::cmp::min;
+use std::collections::HashSet;
+extern crate lazy_static;
+extern crate needletail;
+use needletail::{fastx};
+
+use std::io::{BufReader};
+use std::io::prelude::*;
+use std::fs::File;
+
+fn read_kmer_file(kmer_file: &str) -> HashSet<&'static str> {
+    let mut kmers = HashSet::new();
+
+    // Read in coding k-mers
+    let f = File::open(kmer_file)?;
+    let f = BufReader::new(f);
+
+    for line in f.lines() {
+        let kmer = line.unwrap();
+        kmers.insert(kmer.to_owned());
+    }
+    return kmers;
+}
+
+fn jaccardize(set1: &HashSet<&str>, set2: &HashSet<&str>, verbose: usize) -> f64 {
+    let denominator = min(set1.len(), set2.len());
+    if denominator > 0 {
+        let numerator = set1.intersection(&set2).len();
```

Comment:
```
let numerator_set: HashSet<_> = set1.intersection(&set2).collect();
let numerator = numerator_set.len();
```

Reply:
In this particular case you can do
```suggestion
        let numerator = set1.intersection(&set2).count();
```
and avoid `collect()` (which is going to create a new `HashSet`, and then discard it).

Reply:
nice, thanks @luizirber i was looking for something like that.. much better solution.